docs/guides/faq.md

```diff
@@ -41,8 +41,28 @@ Building a template or demo document is an important part of creating a Pack, as
 Coda supports both document- and page-level icons, from a library we source from an external provider. While some this library includes the icons for some popular brands, it's not comprehensive and your brand may not be present. At the moment we don't have a process for adding new icons to the library, nor do we support using custom icons within a document. This is something we hope to address in the future, but for the time being we recommend using a generic icon.
 
 
+## Can formulas return dynamic schemas?
+
+No, unfortunately they can't. While [dynamic sync tables][dynamic_sync_tables] can flex their schema based upon the specific data source being connected to, this option isn't available for formulas. The schema for a formula must be declared statically, and there is no equivalent `getSchema()` function that formulas can provide.
+
+If you have a formula that returns a dynamic set of custom fields, there are two common approaches:
```

Comment:
some 1pp's set `includeUnknownProperties` to true for the formula schema where the formula may return extra fields. I am not sure who those fields will be used though. might be fine not to mention it here.

Reply:
Ya, I omitted that option since the fields can be seen in the chip hover, but can't be accessed by a formula, shown in a column, etc.